Cargo.toml

```diff
@@ -13,12 +13,15 @@ include = [
     "Cargo.toml",
 ]
 edition = "2018"
+resolver = "2"
```

Comment:
Is the change to resolver v2 needed for this change, or is it a nice to have / drive by cleanup?

reference https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2

Reply:
Actually, it was added to pass CI 'compile-no-std' check

Because the dev-dependency 'simple_logger' introduces the std feature of log, if resolver=2 is not used, libstd will be implicitly introduced in the CI 'compile-no-std'